### ISSUE TITLE
Can't compile getting SteamAuth reference error.

### ISSUE THREAD

**Issue by PachiMc** [Jessecar96/SteamBot#1146]
he type or namespace name 'SteamAuth' could not be found. 2017 vs, nuget actived and i tried reinstalling the package

**Comment by ghost** [comment]
Have you added it as a project ~~dependency~~ reference?

**Comment by PachiMc** [comment]
> Have you added it as a project dependency reference?

u mean active auto download nuget packs?

**Comment by ghost** [comment]
No, with some visual studio projects you have to right click where it says References in the solution manager and add the SteamAuth.dll .

**Comment by PachiMc** [comment]
> No, with some visual studio projects you have to right click where it says References in the solution manager and add the SteamAuth.dll .

thanks, but i dont have a reference called "SteamAuth.dll " to add, only some of .net framework, where can i find it ? image: https://www.screencast.com/t/aExEM5v4Hb

**Comment by ghost** [comment]
The reference exists, it just cannot be found. Can you go to NuGet and install the latest version of SteamAuth (3.0.0) to your project?

Edit: What Antivirus software have you got installed? Some Antivirus software may remove the .dll.

**Comment by alikoza** [comment]
> The reference exists, it just cannot be found. Can you go to NuGet and install the latest version of SteamAuth (3.0.0) to your project?
> 
> Edit: What Antivirus software have you got installed? Some Antivirus software may remove the .dll.

Can you help me for installing and updating this project ? I can pay for it .

**Comment by ghost** [comment]
What issues are you having? If you contact me using my email I will be able to provide better help.

**Comment by alikoza** [comment]
I have contacted with you by email already.  the issue is after updating dependencies bot not works. it seems there is something changed in steam API side. In all cases it gives me an error like warning: you have been disconnected from steam.
